resources/dicts/thoughts/dead/darkforest/general.json

```diff
@@ -51,7 +51,13 @@
             "Feels the shadows watching {PRONOUN/m_c/object}",
             "Wishes to be alone",
             "Misses the feeling of a warm nest",
-            "Feels a sharp pang of hunger"
+            "Feels a sharp pang of hunger",
+            "Dearly misses sunshine",
+            "Misses the warm glow of the moon",
+            "Finds it eerie that this place is so silent",
+            "Wishes to hear songbirds once again",
+            "Hates that the waters are so murky here",
+            "Insists that {PRONOUN/m_c/subject} were in the right"
```

Comment:
you'll want to add a verb tag in place of "were", so that it doesn't end up with things like "she were"

Reply:
completed all of them, hows it looking now?